docs/release_notes/9_5.md

```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
 
-* **SELinux** userspace release 3.6 introduces deny rules for further customizing policies
-* **Keylime** server components, the verifier and registrar, are available as Containers
-* **Rsyslog** processing system introduces customizable TLS/SSL encryption settings and additional options that relate to capability dropping
-* **OpenSSL** TLS toolkit adds a drop-in directory for provider-specific configuration files
-* **libkcapi** 1.4.0 introduces new tools and options. Notably, with the new `-T` option, you can specify target file names in hash-sum calculations
-* **stunnel 5.7.1** TLS/SSL tunneling service changes the behavior of OpenSSL 1.1 and later versions in FIPS mode. Besides this change, this version provides many new features, such as support for modern PostgreSQL clients
+* **SELinux** The SELinux policy now provides a boolean that allows QEMU Guest Agent to execute confined commands 
+* **OpenSSL** TLS toolkit upgrades to 3.2.2. OpenSSL now supports certificate compression and adds in Brainpool curves to the TLS 1.3 protocol.
+* **ca-certificates** program now provides trusted CA roots in the OpenSSL directory format format
+* **crypto-policies** packages are updated to extend its control to algorithm sections in Java
+* **NSS** cryptographic toolkit are rebased to upstream version 3.101
 
 ### Dynamic programming languages, web, and database servers
 
-For a detailed list of the changes in this category see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/new-features#new-features-dynamic-programming-languages-web-and-database-servers).
+For a detailed list of the changes in this category see the [upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23technology-previews-dynamic-programming-languages-web-and-database-servers).
 
 Later versions of the following Application Streams are now available:
 
-* **Python 3.12**
-* **Ruby 3.3** available as a module stream
-* **PHP 8.2** available as a module stream
-* **nginx 1.24** available as a module stream
-* **MariaDB 10.11** available as a module stream
-* **PostgreSQL 16** available as a module stream
+* **Apache HTTP Server 2.4.62**
+* **Node.js 22**
+
+### Compilers and development tools
 
-The following applications have been upgraded:
+For details on the changes in this category [see the upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23new-features-compilers-and-development-tools)
```

Comment:
HTML: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.5_release_notes/new-features#new-features-compilers-and-development-tools

Reply:
Think you took the wrong line but also added this 👍🏻